### ISSUE TITLE
Improvements to "From the Ground Up" example

### ISSUE THREAD

**Issue by tkuchida** [opensim-org/opensim-core#1775]
[From the Ground Up: Building a Passive Dynamic Walker Model](http://simtk-confluence.stanford.edu:8080/display/OpenSim/From+the+Ground+Up%3A+Building+a+Passive+Dynamic+Walker+Model)

- [ ] Provide initial conditions that produce a couple steps of walking for a sense of satisfaction after correctly building the model.
- [x] The instruction "create a text file" should be more specific to avoid creating "DynamicWalkerBuildModel.cpp.txt" with Notepad++ (e.g., "create a new file and select *Save As…* in Notepad++ to save as a .cpp file").
- [x] `CoordinateSet& platformJoints = ...` should be `CoordinateSet& platformCoordinates = ...`.
- [x] It is unclear how to set the location_in_parent and location_in_child properties. Add explanation and figure.
- [x] Move example files into the opensim-core repo.
- [x] Update doxygen links for OpenSim (won't be able to do this until after the release, when we post OpenSim's doxygen).
- [x] Upload the comparison .mot file as an attachment to the confluence page.

**Comment by chrisdembia** [comment]
I've addressed two of the easier tasks above.

**Comment by chrisdembia** [comment]
Creating good initial conditions is taking longer than I had hoped, so I might push changes to the example without good initial conditions and add that later.

**Comment by tkuchida** [comment]
I clarified some of the instructions and added an "info" box on [this page](https://simtk-confluence.stanford.edu:8443/display/OpenSim/Part+II%3A+Building+a+Passive+Dynamic+Walker+Model) (in Part A, under the "Setting the Properties of the Coordinates in the Pin Joint" heading) to explain the difference between "get…" and "upd…" methods, which is a common source of confusion. @jimmyDunne It would be nice if we could find initial conditions that cause the model to do something (anything) more interesting than just falling.

**Comment by jenhicks** [comment]
I looked around for some initial conditions from the files Dan Jacobs put together originally. I couldn't find anything unfortunately.

**Comment by chrisdembia** [comment]
> I clarified some of the instructions

👍 

> It would be nice if we could find initial conditions that cause the model to do something (anything) more interesting than just falling.

I spent a good amount of time trying to get good initial conditions using an optimizer, and @aseth1 shared some files also. It's hard to get anything other than falling, especially if not adding "magnets" to the knee.

**Comment by tkuchida** [comment]
Anything? Could we lock the knees and hips and make the model bounce down the ramp?

**Comment by tkuchida** [comment]
> Perhaps the user could be instructed to lock the knees in the GUI before simulating.

Yes, that is what I meant.

**Comment by chrisdembia** [comment]
oh oops okay. I can look into it.

**Comment by tkuchida** [comment]
Could include a script that adds feet, if that's the issue. Something like what [this YouTube video](https://youtu.be/lZqQUYw7h3k?t=31s) shows would be cool.

**Comment by chrisdembia** [comment]
Here's my proposed change to the example:

Before:

> D. Simulate Your Model
We have finished writing the code to build the model. Compile your code and print your model. Load the model in the OpenSim GUI and run a forward simulation from 0 to 1 seconds. Make sure your model is in the default pose before running the forward simulation. Use the motion file 'DynamicWalkerForward.mot' (located in the downloaded ZIP file) to compare the results to those of your model.

After:

> D. Simulate Your Model
We have finished writing the code to build the model. Compile your code and print your model. Load the model in the OpenSim GUI and run a forward simulation from 0 to 1 seconds. Make sure your model is in the default pose before running the forward simulation. To ensure you built the model correctly, use the motion file 'DynamicWalkerForward.mot' (located in the downloaded ZIP file) to compare the results to those of your model.
> The DynamicWalkerForward.mot motion is not very interesting, though. Let's have some fun! Here are some settings to try before simulating:
> - Turn the model into two double pendulums by locking pelvis_tx and pelvis_ty.
> - Make the model do a flip by locking the hip and knee joints and setting the initial speed of the pelvis_ty coordinate to a large negative number.
